config/development.json

```diff
@@ -0,0 +1,6 @@
+{
+  "io_sites": {
+    "max_static_paths": 0,
+    "revalidate": 360
+  }
+}
```

Comment:
I'm not sure why I prefer snake_case for configs, but it's a strong opinion loosely held so happy to convert to camelCase 😄

Reply:
+1, also loosely held, I think I just dig the `CONST` energy 💯